docs/core/extensions/logging-providers.md

```diff
@@ -80,11 +80,6 @@ The `Console` provider logs output to the console.
 
 The `Debug` provider writes log output by using the <xref:System.Diagnostics.Debug?displayProperty=fullName> class, specifically through the <xref:System.Diagnostics.Debug.WriteLine%2A?displayProperty=nameWithType> method. The <xref:Microsoft.Extensions.Logging.Debug.DebugLoggerProvider> creates <xref:Microsoft.Extensions.Logging.Debug.DebugLogger> instances, which are implementations of the `ILogger` interface.
```

Comment:
could you please add sentence here that this only happen when the debugger is attached?